doc/sphinx/explanation/performance.md

```diff
@@ -0,0 +1,59 @@
+# Performance
+
+This document aims to explore in depth the performance considerations around Mir
+based compositors.
+
+Note that Mir is a C++ library for building compositors, not a product itself.
+The following applies to all Mir based compositors, but it is theoretically
+possible to defeat the available performance in such products.
+
+## Memory usage
+
+Memory usage varies a lot depending on: 
+
+* the GPU characteristics of the system
+* the number, size and configuration of the displays
+* the number of application windows being composited
+* the rate at which buffers are submitted to the application windows
```

Comment:
This block can probably be moved to the top of the document.

Reply:
Only the last three lines of the block are common

Reply:
I agree it would read better if the common lines here were abstracted up.